jenkins/gce-custom-images/prepare_custom_leonardo_gce_image.sh

```diff
@@ -26,7 +26,7 @@ terra_jupyter_r="us.gcr.io/broad-dsp-gcr-public/terra-jupyter-r:0.0.11"
 terra_jupyter_bioconductor="us.gcr.io/broad-dsp-gcr-public/terra-jupyter-bioconductor:0.0.12"
 terra_jupyter_gatk="us.gcr.io/broad-dsp-gcr-public/terra-jupyter-gatk:0.0.13"
 terra_jupyter_aou_old="us.gcr.io/broad-dsp-gcr-public/terra-jupyter-aou:1.0.14"
-terra_jupyter_aou="us.gcr.io/broad-dsp-gcr-public/terra-jupyter-aou:1.0.18"
+terra_jupyter_aou="us.gcr.io/broad-dsp-gcr-public/terra-jupyter-aou:1.0.20"
```

Comment:
@calbach FYI Leo images will now have cached:
```
aou = 1.0.20
aou_old = 1.0.14
```
AFAICT AoU prod is still on `1.0.14` -- let me know if you have any concerns.

Reply:
Correct, looks good.